src/Rules/Doctrine/ORM/EntityColumnRule.php

```
@@ -0,0 +1,94 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Doctrine\ORM;
+
+use PhpParser\Node;
+use PHPStan\Analyser\Scope;
+use PHPStan\Reflection\MissingPropertyFromReflectionException;
+use PHPStan\Rules\Rule;
+use PHPStan\Type\Doctrine\DescriptorRegistry;
+use PHPStan\Type\Doctrine\ObjectMetadataResolver;
+use PHPStan\Type\TypeCombinator;
+use PHPStan\Type\VerbosityLevel;
+use Throwable;
+use function sprintf;
+
+class EntityColumnRule implements Rule
+{
+
+	/** @var \PHPStan\Type\Doctrine\ObjectMetadataResolver */
+	private $objectMetadataResolver;
+
+	/** @var \PHPStan\Type\Doctrine\DescriptorRegistry */
+	private $descriptorRegistry;
+
+	public function __construct(ObjectMetadataResolver $objectMetadataResolver, DescriptorRegistry $descriptorRegistry)
+	{
+		$this->objectMetadataResolver = $objectMetadataResolver;
+		$this->descriptorRegistry = $descriptorRegistry;
+	}
+
+	public function getNodeType(): string
+	{
+		return Node\Stmt\PropertyProperty::class;
+	}
+
+	/**
+	 * @param \PhpParser\Node\Stmt\PropertyProperty $node
+	 * @param \PHPStan\Analyser\Scope $scope
+	 * @return array<\PHPStan\Rules\RuleError|string>
+	 */
+	public function processNode(Node $node, Scope $scope): array
+	{
+		$class = $scope->getClassReflection();
+		if ($class === null) {
+			return [];
+		}
+
+		$objectManager = $this->objectMetadataResolver->getObjectManager();
+		if ($objectManager === null) {
+			return [];
+		}
+
+		try {
+			$metadata = $objectManager->getClassMetadata($class->getName());
+		} catch (Throwable $e) {
+			return [];
```

Don't silence all errors :)